client/payment-details/summary/index.tsx

```diff
@@ -3,7 +3,7 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
```

Comment:
The `sprintf` is not utilized in the file, it should be removed.

Reply:
Indeed, fixed 85e7afefb